Publish /fr/lecons/visualisations-interactives-plotly

anisa-hawes opened the pull request:
Preparing files for publication.

Closes #3639 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [x] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Manager @anisa-hawes 
- [x] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [x] Assign at least one individual or team to "Reviewers"
  - [ ] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.

anisa-hawes committed:
Update ph_authors.yml

Add FR bio for Grace Di Méo.

anisa-hawes committed:
Upload fr-tr-interactive-visualization-with-plotly figures

Upload FR figures.

anisa-hawes committed:
Create visualisations-interactives-plotly.md

Create Markdown file for new FR translation.

anisa-hawes committed:
Update visualisations-interactives-plotly.md

- Add `translation_date:`
- Add `doi:`

Commit:
Merge branch 'gh-pages' into publish-visualisations-interactives-plotly

Commit:
Merge branch 'publish-visualisations-interactives-plotly' of github.com:programminghistorian/jekyll into publish-visualisations-interactives-plotly

anisa-hawes commented:
Bonjour @marie-flesch,

I’ve staged visualisations-interactives-plotly for publication. You can [read it through in Preview](https://deploy-preview-3640--ph-preview.netlify.app/fr/lecons/visualisations-interactives-plotly) and let me know if you notice anything which you'd like to adjust. 

N.B. I've double-checked each of the interactive plots (via the links in the figure captions) and I am satisfied with these. Let me know if you spot any problems?

May I also ask if you could check my translation of the original author's bio?

**EN** `Grace Di Méo is a Lecturer in Criminology in the Faculty of Humanities and Social Sciences at Oxford Brookes University.`

**FR** `Grace Di Méo est maître de conférences en criminologie à la faculté des sciences humaines et sociales de Oxford Brookes University.`

Let's aim to publish this next Wednesday 24th September ☺️ 
Thank you !

marie-flesch commented:
Here's a new version of the bio: 
> Grace Di Méo est maitresse de conférences en criminologie à la faculté des sciences humaines et sociales d'Oxford Brookes University.

marie-flesch commented:
all interactive figures look great!

marie-flesch commented on fr/lecons/visualisations-interactives-plotly.md:
just a small change here: remove the capital letter at "Exporter".
So: "Afficher et exporter les visualisations"

marie-flesch commented:
It looks really good! Just a minor change in addition to the bio. I'll approve the review when you have time to make them. Thank you so much @anisa-hawes !

anisa-hawes committed:
Update visualisations-interactives-plotly.md

Lowercase exporter, line 1124

anisa-hawes committed:
Update ph_authors.yml

Correct Grace's bio, line 2992.

Commit:
Merge branch 'gh-pages' into publish-visualisations-interactives-plotly

anisa-hawes commented:
Many thanks, @marie-flesch.

I've made those adjustments:

- Removed the uppercase letter E so that the section heading reads: **Afficher et exporter** [...] https://github.com/programminghistorian/jekyll/pull/3640/commits/fa4cab4c96c4f9b089db7537e44d249d32b618a8
- Adjusted my translation of the author's bio with your corrections (thank you!) : https://github.com/programminghistorian/jekyll/pull/3640/commits/237000e998730b35c3c1ce0aafd1ec7fbe7942fa

If you have a moment to add your review, the lesson is now ready.

Following publication, I'll update you as soon as the DOI is live and share links to our announcements on social media.

marie-flesch reviewed (approved):
all good!
